bdk_chain/src/chain_graph.rs

```diff
@@ -116,6 +131,21 @@ impl<P: ChainPosition> ChainGraph<P> {
         Ok(changeset)
     }
 
+    /// Given a transaction, return an iterator of conflicting [`Txid`]s that are marked within the
+    /// internal chain.
+    pub fn conflicting_txids<'a>(
+        &'a self,
+        tx: &'a Transaction,
+    ) -> impl Iterator<Item = (&'a P, Txid)> + 'a {
+        self.graph
+            .conflicting_txids(tx)
+            .filter_map(|(_, conflicting_txid)| {
+                self.chain
+                    .tx_position(conflicting_txid)
+                    .map(|conflicting_pos| (conflicting_pos, conflicting_txid))
+            })
+    }
```

Comment:
I might be confused, but are we saying, we will only find conflicts if they are both in graph and chain? What happens for conflicts which are **only in graph** and **not in chain**? if we don't detect them, should we clarify that in the comment?

Reply:
We don't care about conflicts that are in the graph but not the chain. The chain is the thing that must be consistent. This is a crucial concept to understand when using the APIs and it should be very well documented but we don't need comments all over the place every time we employ this fact.